Sources/SafeDICore/Models/Scope.swift

```diff
@@ -80,68 +80,100 @@ final class Scope: Hashable {
         propertyStack: OrderedSet<Property>,
         erasedToConcreteExistential: Bool
     ) throws -> ScopeGenerator {
-        if
```

Comment:
Best reviewed with whitespace ignored.

The check in this `if` statement was moved (with changes) below under the comment `// Check received children for cycles.`